src/Redis/Indexes/Builders/OneToOneIndexBuilder.php

```diff
@@ -129,7 +129,7 @@ protected function getSysKey(IndexDto $dto) : string
      */
     protected function getSysField(IndexDto $dto) : string
     {
-        return $this->keys->makeSysField($dto->docId, $dto->docType);
+        return $this->keys->makeSysField($dto->docId, $dto->name);
```

Comment:
а тут разве не должно быть 
```suggestion
        return $this->keys->makeSysField($dto->docId, $dto->docType, $dto->name);
```

Reply:
@steepdevs $dto->docType Используется уже в генерации ключа getSysKey Мне кажется будет избыточно использовать его ещё и в названии поля